### PR TITLE
Use the base image (gclub/skywardai) instead and change the working directory to /app.

### LINKED ISSUE #57: Some common and rarely modified build processes should be implemented in the base image.
Some common and rarely modified build processes should be implemented in the base image.
For example, the tools and python modules that the program depends on.
After this change, the program build is performed directly on the base image, reducing the build steps and build time.

### PR DESCRIPTION
**Pull request by arvin936**
Use the base image (gclub/skywardai) instead and change the working directory to /app.

**Description**

Use base images to reduce build steps and build time.
The Image working directory is unified into the /app directory that is commonly used in open source projects.
This PR fixes #57

**Notes for Reviewers**

**[Signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -1,35 +1,20 @@
...
-RUN pip3 install -r requirements.txt
-
 # Copy all files
 COPY . .
 
-# Copy entrypoint.sh for auto connection with account_db service
-COPY ./entrypoint.sh .
-RUN chmod +x /usr/backend/entrypoint.sh
+# Add execution permissions to entrypoint.sh
+RUN chmod +x /app/entrypoint.sh
```
Comment (Aisuko) [inline backend/Dockerfile:14]: As we discussed, could we leave a correct permission to the entrypoint.sh. So, we can copy it to container directly.

**Review by Aisuko — CHANGES_REQUESTED** [review]

File: backend/Dockerfile (hunk 1 above)
Comment (Aisuko) [reply backend/Dockerfile:14]: Here is another request see https://github.com/SkywardAI/containers/issues/1

**Review by Aisuko — COMMENTED** [review]
We should fix the security issue of the base image, and then merge this one.

**Review by Micost — APPROVED** [review]

**Comment by Aisuko** [comment]
@Micost @arvin936 As we discussed, we should remove this chmod. It will not be supported in multiple-platforms build.

**Comment by Aisuko** [comment]
Close by #72

**Pull request closed without merging**